service/src/main/resources/db/changesets/20241106-base-data-insertion.yaml

```diff
@@ -0,0 +1,239 @@
+# this is a consolidated yaml that contains all the data initially inserted in the database
+databaseChangeLog:
+  - changeSet:
+      id: consolidated-schema
+      author: js
+      changes:
+        # Insert initial array_imputation pipeline data
+        - insert:
+            tableName: pipelines
+            columns:
+              - column:
+                  name: name
+                  value: array_imputation
+              - column:
+                  name: version
+                  value: 0
+              - column:
+                  name: display_name
+                  value: Array Imputation
+              - column:
+                  name: description
+                  value: Phase and impute genotypes using Beagle 5.4 with the AoU/AnVIL reference panel of 515,579 samples.
+              - column:
+                  name: pipeline_type
+                  value: imputation
+              - column:
+                  name: wdl_url
+                  value: https://github.com/broadinstitute/warp/blob/TSPS-183_mma_beagle_imputation_hg38/pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl
```

Comment:
wasnt sure if this was correct but i think this is the latest value we have

Reply:
i think that's right too (it matches what we currently have)